notifications.bs

```diff
@@ -426,6 +456,8 @@ interface Notification : EventTarget {
   static readonly attribute NotificationPermission permission;
   [Exposed=Window] static void requestPermission(optional NotificationPermissionCallback callback);
 
+  static readonly attribute unsigned long maxActions;
```

Comment:
@sicking also suggested this in #44. Availability of actions might influence how a page emphasizes alternatives to actions on the page displayed in response to activating the notification.

That said, I'm having mixed feelings about only storing `maxActions` actions in the action list.